runtime/queries/prql/injections.scm

```diff
@@ -0,0 +1,11 @@
+(
+ (s_string) @sql
+ (#offset! @sql 0 2 0 -1)
+)
+
+(from_text
+  (keyword_from_text)
+  (keyword_json)
+  (literal) @json
+  (#offset! @json 0 3 0 -3)
+)
```

Comment:
```suggestion
((s_string) @injection.content
 (#set! injection.language "sql"))

(from_text
  (keyword_from_text)
  (keyword_json)
  (literal) @injection.content
  (#set! injection.language "json"))
```

We use a slightly different syntax for injections as well: https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection